### DISCUSSION TITLE
large folders in .cache/vscode-cpptools

### DISCUSSION THREAD

**Discussion by maziar840** [microsoft/vscode-cpptools#12488] [Q&A] [answered]
I have a limited quota and large folders in ~/.cache/vscode-cpptools that look like this:

![image](https://github.com/user-attachments/assets/76f48438-b8d4-4859-b7a0-4b66fbe594c6)

I have already set the Intelli Sense Cache Path to somewhere else, but these are still generated in that folder (after I delete them). Any ideas what they are and how to prevent them from being regenerated there?

**Comment by sean-mcmanus** [comment, marked as answer]
@maziar840 You also have to set C_Cpp.default.browse.databaseFilename to something else, such as `"${workspaceFolder}/.vscode/vc.db"`.

**Reply by maziar840** [reply]
Seems to be working. Thank you.

**Reply by LiZhenYu2000** [reply]
I set `C_Cpp.default.browse.databaseFilename` and `C_Cpp.intelliSenseCachePath` to something else, but these dirs are still regenerated after being deleted, even if no file within these dirs. 
Is there any way to completely prevent these dirs from being regenerated ?
